### DISCUSSION TITLE
How safe is this method?

### DISCUSSION THREAD

**Discussion by yo-less** [007revad/Synology_enable_M2_volume#202] [Q&A] [answered]
First of all, thanks a lot for giving us the option to use the SSDs in our NAS to create an additional storage pool. I have some docker containers that are extremely slow and that are said to be much faster running from SSDs. I'm a bit worried that I might lose data if I transfer my docker setup to an SSD storage pool that isn't officially supported and that gets removed every update. If I run Synology_enable_M2_volume on every shutdown and Synology_HDD_db on every startup I shouldn't really have to deal with any additional maintenance and I could be pretty sure that my volumes would always be available and working fine? I understand that nobody can guarantee anything, but I'm just wondering how much risk is involved in running your scripts?

**Comment by 007revad** [comment, marked as answer]
Which model Synology do you have? Many '20 series and newer models don't need Synology_enable_M2_volume (they only need Synology_HDD_db).

Regardless of whether you run the script, or scripts, at shutdown or bootup you'd still need to reboot after a DSM major update.

Moving the docker shared folder to the NVMe volume is fine.

**Comment by yo-less** [comment]
Thanks for your fast reply, I have a DS920+ and will take the plunge then and try it out. I'll try and set it up with Synology_HDD_db only. Thanks again!
